modules/nextflow/src/main/groovy/nextflow/datasource/SraExplorer.groovy

```diff
@@ -43,7 +53,9 @@ import nextflow.util.Duration
 @Slf4j
 class SraExplorer {
 
-    static public Map PARAMS = [apiKey:[String,GString], cache: Boolean, max: Integer, protocol: ['ftp','http','https']]
+    static public Map PARAMS = [apiKey:[String,GString], cache: Boolean, max: Integer, protocol: ['ftp','http','https'],
+                                retryPolicy: Map]
+    final static List<Integer> RETRY_CODES = List.of(408, 429, 500, 502, 503, 504)
```

Comment:
```suggestion
    final static public List<Integer> RETRY_CODES = List.of(408, 429, 500, 502, 503, 504)
```

better adding the access visibility ie. `public`, otherwise groovy will generate a getter method for it